### PR TITLE
[healthcare] fix: fix broken tests for healthcare API

### LINKED ISSUE #3120: healthcare.api-client.hl7v2.hl7v2_messages_test: test_CRUD_hl7v2_message failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 1037f47ef34fa1576402e5391cf62910468e31c9
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/e99e8778-ca3a-432c-a825-2c20b711080c), [Sponge](http://sponge2/e99e8778-ca3a-432c-a825-2c20b711080c)
status: failed

### LINKED ISSUE #3134: healthcare.api-client.hl7v2.hl7v2_messages_test: test_patch_hl7v2_message failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 373789eb5587aef960f9b029ed5034c35b37fd22
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/f92341e8-14a1-46d4-9a77-7cd8bc505824), [Sponge](http://sponge2/f92341e8-14a1-46d4-9a77-7cd8bc505824)
status: failed

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #3120
fixes #3134

### CODE REVIEW

**Comment by tmatsuo** [comment]
It seems like the API changed a bit, so I needed to make some tweak. I added @busunkim96 and @leahecole as reviewers, but let me know if I'm doing wrong.

**Comment by tmatsuo** [comment]
Wow. It seems like those 2 builds are using the same resource name. I think this is a recipe for disaster. I'll also fix it in the next commit.

File: healthcare/api-client/hl7v2/hl7v2_messages_test.py [hunk 1]
```diff
@@ -28,8 +29,14 @@
 project_id = os.environ['GOOGLE_CLOUD_PROJECT']
 service_account_json = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
 
-dataset_id = 'test_dataset_{}'.format(int(time.time()))
-hl7v2_store_id = 'test_hl7v2_store-{}'.format(int(time.time()))
+# assume we can access the OS provied raondom source.
```
Comment (busunkim96) [inline healthcare/api-client/hl7v2/hl7v2_messages_test.py:32]: Other samples generally use [uuid](https://docs.python.org/3/library/uuid.html) module to generate a random string for resource names. 

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/fe705fc42371d7f9697b2ef3192bc59008d7b136/dialogflow/cloud-client/create_document_test.py#L18-L28

Comment (tmatsuo) [reply healthcare/api-client/hl7v2/hl7v2_messages_test.py:32]: Certainly we can use uuid, but having the time portion in the string has a large benefit, especially for resources which can leak. We can safely delete old leaked resources manually, or even automatically.

Leaked resources are potentially sources of test flakiness, so I'd stick to having the time portion.

Thoughts?

**Comment by tmatsuo** [comment]
It's still flaky:

```
Traceback (most recent call last):
  File "/tmpfs/src/github/python-docs-samples/healthcare/api-client/hl7v2/hl7v2_messages_test.py", line 98, in test_CRUD_hl7v2_message
    hl7v2_message_name = hl7v2_messages_list[0].get('name')
IndexError: list index out of range
```

As far as I read the (log)[https://source.cloud.google.com/results/invocations/3cdf88e2-52f8-47e0-a757-a2792429c141/log], the test successfully created a new message and the subsequent list call returned an empty result.

Maybe I can retry for several times.

File: healthcare/api-client/hl7v2/hl7v2_messages_test.py (hunk 1 above)
Comment (tmatsuo) [reply healthcare/api-client/hl7v2/hl7v2_messages_test.py:32]: TIL uuid has time field, so yeah uuid is fine

**Comment by tmatsuo** [comment]
@busunkim96 I ran the tests locally 10 times and all passed. PTAL

**Comment by tmatsuo** [comment]
Adding `.gitignore` causes the builds running all the tests :( I'll revert it

**Comment by leahecole** [comment]
@noerog PTAL - it's a LGTM from me, but given that you're the codeowner, I just want your final stamp :)

**Comment by noerog** [comment]
LGTM, thank you @tmatsuo

**Pull request merged**